### PR TITLE
ssh-vault 0.12.11

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ==> go build -trimpath -o=/home/linuxbrew/.linuxbrew/Cellar/ssh-vault/0.12.11/bin/ssh-vault -ldflags=-s -w -X main.version=0.12.11 cmd/ssh-vault/main.go
  package cmd/ssh-vault/main.go is not in std (/home/linuxbrew/.linuxbrew/Cellar/go/1.21.4/libexec/src/cmd/ssh-vault/main.go)
```

File: Formula/s/ssh-vault.rb [hunk 1]
```diff
@@ -18,18 +18,29 @@ class SshVault < Formula
...
-  depends_on "go" => :build
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "openssl@3"
 
   def install
-    ldflags = "-s -w -X main.version=#{version}"
-    system "go", "build", *std_go_args(ldflags: ldflags), "cmd/ssh-vault/main.go"
+    # Ensure that the `openssl` crate picks up the intended library.
+    ENV["OPENSSL_DIR"] = Formula["openssl@3"].opt_prefix
```
Comment (SMillerDev) [inline Formula/s/ssh-vault.rb:27]: Can we check for linkage with this?

**Comment by nbari** [comment]
should be fixed with this https://github.com/Homebrew/homebrew-core/pull/154122

**Comment by chenrui333** [comment]
#154122

File: Formula/s/ssh-vault.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/s/ssh-vault.rb:27]: we did in #154122, good call. 👍

**Pull request closed without merging**